src/store/to-do/sagas.js

```diff
@@ -11,6 +13,18 @@ export function* getToDos() {
   }
 }
 
+export function* deleteToDoItem({ payload }) {
+  try {
+    yield call(toDoService.deleteItem, payload);
+    history.push(ROUTES.DASHBOARD);
```

Comment:
moze i odma u komponenti redirect, ne moras uvek u sagi, u sagi je ok ako bas moras, ovako ja ne bih iz redux middleware-a to radio. Ne moras da cekas u sagi da li je proslo ili ne, znas da je kliknuo da obrise neka ga redirektuje

Reply:
Trebala bi mozda iz store-a da izbacis ako ne fetchujes opet podatke, ako fetchujes onda nemas problem :)